apps/step-function.json.j2

```diff
@@ -1,6 +1,14 @@
 {
-  "StartAt": "JOB_RUNNING",
+  "StartAt": "SET_DEFAULT_RESULTS",
   "States": {
+    "SET_DEFAULT_RESULTS": {
```

Comment:
Not blocking, but I think I'd prefer `JOB_RUNNING` still be the first task, since we want to update the status code in the database from PENDING to RUNNING as quickly as possible. No real risk in this case, but putting other tasks in front of JOB_RUNNING might cause us to lose sight of it down the road.

Reply:
I could easily make `SET_DEFAULT_RESULTS` be the second state if it only needs to set one field in `$.results`, but if it ever needs to set multiple fields, then I'm not sure how to do that without overwriting the `results` object, given the [documentation](https://states-language.net/spec.html#filters):

> If the raw input has a field at the location addressed by the ResultPath value then in the output that field is discarded and overwritten by the state's result.

So it seems necessary for `SET_DEFAULT_RESULTS` to create `$.results` before `JOB_RUNNING` creates `$.results.job_running`. Unless I'm misunderstanding how `ResultPath` works and it would actually just create the missing fields without overwriting the entire object?

Reply:
That checks out, no changes necessary, then :+1: